sdcm/snitch_configuration.py

```diff
@@ -58,4 +58,7 @@ def apply(self) -> None:
             properties['dc_suffix'] = self._dc_suffix
 
         with self._node.remote_cassandra_rackdc_properties() as properties_file:
+            if properties_file.get('dc') is not None:
+                # in case when dc is already set (e.g. it is forced by AddRemoveDc nemesis), skip setting it
+                del properties['dc']
```

Comment:
maybe do it a bit more general for all of the props ? with `dict.setdefault` ? and not just for `dc` ?

Reply:
```python
for k,v in properties.items:
   properties_file.setdefault(k, v)
```